### PR TITLE
Update get-help-support.md

### PR DESCRIPTION
**Pull request by msbemba**
made slight changes to bring the drop-down list immediately below the phone support heading.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
@yogkumgit  [cmcatee-MSFT](https://github.com/cmcatee-MSFT) is the author for this document.

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8047

File: microsoft-365/admin/get-help-support.md [hunk 1]
```diff
@@ -54,8 +54,11 @@ Save time by starting your service request online. We'll help you find a solutio
 
 ## Phone support
 
-Billing support is provided in English from 9 AM-5 PM (9 AM-6 PM in Australia), Monday-Friday.\
-Technical support is provided in English 24 hours a day, 7 days a week.\
+Use the drop-down menu below to find the number for your country or region.
+
+[!INCLUDE [country-region-support-dropdown-list](../admin/includes/country-region-support-dropdown-list.md)]
+
+Billing support and Technical support hours (and languages) are listed on each Country page selected from the drop down.\
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/get-help-support.md]:
```suggestion
Billing support and Technical support hours (and languages) are listed on each country's page selected from the drop-down.
```

File: microsoft-365/admin/get-help-support.md [hunk 2]
```diff
@@ -54,8 +54,11 @@ Save time by starting your service request online. We'll help you find a solutio
...
 Admins, have your account details ready when you call.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/get-help-support.md:63]:
```suggestion

Admins, have your account details ready when you call.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@efrene Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@tiburd Please help

**Review by efrene — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**